package.json

```diff
@@ -111,7 +111,7 @@
     "ember-inflector": "^2.0.0"
   },
   "engines": {
-    "node": ">= 0.12.0"
```

Comment:
FWIW, in practice we have already done this bump (we include deps that are only Node 4+ compatible since at least 2.12).